src/Elmish.WPF/ViewModel.fs

```diff
@@ -582,7 +572,7 @@ and [<AllowNullLiteral>] internal ViewModel<'model, 'msg>
         match binding with
         | TwoWay (_, set)
         | TwoWayValidate (_, set, _) ->
-            dispatch <| set value currentModel
+            currentModel |> set value |> dispatch
```

Comment:
Please change to `set value currentModel |> dispatch` or leave as it was (cf. previous comment on multi-parameter functions and pipelines)

Reply:
Sure, I did your first change suggestion.